pyspectral/tests/test_blackbody.py

```diff
@@ -53,75 +53,78 @@ def __call__(self, dsk, keys, **kwargs):
         return dask.get(dsk, keys, **kwargs)
 
 
-class TestBlackbody(unittest.TestCase):
+class TestBlackbody:
     """Unit testing the blackbody function."""
 
     def test_blackbody(self):
         """Calculate the blackbody radiation from wavelengths and temperatures."""
         wavel = 11. * 1E-6
         black = blackbody((wavel, ), [300., 301])
-        self.assertEqual(black.shape[0], 2)
-        self.assertAlmostEqual(black[0], RAD_11MICRON_300KELVIN)
-        self.assertAlmostEqual(black[1], RAD_11MICRON_301KELVIN)
+        assert black.shape[0] == 2
+        np.testing.assert_almost_equal(black[0], RAD_11MICRON_300KELVIN)
```

Comment:
the pytest way of doing this is
`assert black[0] == pytest.approx(RAD_11MICRON_300KELVIN)`
but it's fine as it is.

Reply:
These and few others adjusted and some duplicates removed in https://github.com/pytroll/pyspectral/pull/206/commits/30a9aedc1c08aa998d81732ba8dc28e9d6a0a8ec